### ISSUE TITLE
Doesn't evaluate all methods

### ISSUE THREAD

**Issue by paidi** [saeyslab/moduledetection-evaluation#4]
The paper evaluates a large number of different methods, but it looks like the code provided here can only reproduce the evaluations for `dummy` and `agglom`. Is this the case?

**Comment by paidi** [comment]
Thanks for pushing the other methods. One of those commits removed the `conf` folder from the repo, which is required to run the notebooks.

**Comment by zouter** [comment]
Woops sorry, that was unintended. I added them back!

I have to note that even though the wrappers are there, they will likely require you to install some dependencies (R, python or command line). They may also no longer work after 2 years... It was before I discovered docker containers :slightly_smiling_face: . I can probably help if you encounter some problems with specific wrappers, but can't guarantee that all wrappers will still work, sorry about that.

Hope this helps you!
